services/src/main/java/org/jboss/windup/web/services/model/MigrationProject.java

```diff
@@ -33,6 +35,7 @@
  * @author <a href="http://ondra.zizka.cz/">Ondrej Zizka, zizka@seznam.cz</a>
  */
 @Entity
+@Table(uniqueConstraints = @UniqueConstraint(columnNames = "title", name = "uniqueTitle"))
```

Comment:
Right, it will show as "Database constraint was not met". The DB exception doesn't give much more useful message. We would have to look up the existing title to get a specific message.